lib/Command/Classify.php

```diff
@@ -107,6 +137,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 					MusicnnClassifier::MODEL_NAME => [],
 				];
 				foreach ($this->storageService->getFilesInMount($mount['storage_id'], $mount['override_root'], $models, $lastFileId) as $file) {
+
+					if ($pathMatcher !== null) {
```

Comment:
This will go through all files of the nextcloud instance even if only a single folder is given via the parameter. If we check the users of the mounts the we got in line 123 we can short circuit mounts that can not be accessed by the user that was requested. I hope that's understandable :)